src/wp-includes/rest-api/endpoints/class-wp-rest-font-faces-controller.php

```diff
@@ -916,8 +916,8 @@ protected function relative_fonts_path( $path ) {
 		$new_path = $path;
 
 		$fonts_dir = wp_get_font_dir();
-		if ( str_starts_with( $new_path, $fonts_dir['path'] ) ) {
-			$new_path = str_replace( $fonts_dir, '', $new_path );
+		if ( str_starts_with( $new_path, $fonts_dir['basedir'] ) ) {
+			$new_path = str_replace( $fonts_dir['basedir'], '', $new_path );
```

Comment:
Noting that `WP_REST_Font_Faces_Controller::relative_fonts_path ` is based on `_wp_relative_upload_path`, which also uses `basedir` rather than `path`.

I believe using `basedir` here is what was originally intended.